### DISCUSSION TITLE
I keep getting this error with my raspberry pi pico w

### DISCUSSION THREAD

**Discussion by JuicyC2480** [thonny/thonny#3061] [General]
Hello,
Here is the error I keep receiving:
PROBLEM IN THONNY'S BACK-END: Internal error (serial.serialutil.SerialException: WriteFile failed (PermissionError(13, 'The device does not recognize the command.', None, 22))).
See Thonny's backend.log for more info.
You may need to press "Stop/Restart" or hard-reset your MicroPython device and try again.

Process ended with exit code 0.

Here is what I have tried:
- I have double checked my COM port number is correct
- I have checked the backend.log and don't see any big issues
- I have unistalled and reinstalled Thonny
- I have unistalled and reinstalled micropython onto my raspberry pi pico
- Most importantly, I have turned my computer on and off a number of times

Any other ideas for help?

**Comment by aivarannamaa** [comment]
Can you get your device's MicroPython REPL using Putty?
